sites/public/src/forms/applications/FormSummaryDetails.tsx

```diff
@@ -327,6 +327,19 @@ const FormSummaryDetails = ({
               </Fragment>
             ))}
           </ViewItem>
+          {application.programs
```

Comment:
I think it'd be better to switch this to using `.reduce` instead of `.filter` and then `.map`

Reply:
the end result of the maps is an array of arrays containing jsx which is the accumulated value

Ultimately this is an efficiency change though, so I'm okay with leaving this in place for now and if we run into efficiency concerns later we can make this change then